### ISSUE TITLE
GQLAlchemy can't be installed on Windows with Python 3.11

### ISSUE THREAD

**Issue by katarinasupe** [memgraph/gqlalchemy#290]
Due to its dependency on mgclient GQLAlchemy can't be installed on Windows with Python 3.11

**Comment by arunmanoharan4** [comment]
Hi Katrina,

Any ETA on this bug? It will really help us out.

**Comment by katarinasupe** [comment]
Hi @arunmanoharan4, this hasn't been a priority for us so far, and there have been some blockers to adding this support due to some dependencies on GQLAlchemy. Is there a possibility of you using Python < 3.11? Is this blocking your usage of Memgraph?

**Comment by yakuzadave** [comment]
Yeah, I have been running into this issue as well (which is a shame because GQLAlchemy is used for some of my LLM tools as well that work with memgraph), but I have been getting by with the `neo4j` driver on Windows in the meantime.  I'll still want to keep an eye on this and do some testing .

**Comment by katarinasupe** [comment]
Should be resolved with https://github.com/memgraph/pymgclient/issues/66
